src/main/kotlin/ca/kittle/plugins/Databases.kt

```diff
@@ -39,9 +42,20 @@ fun Application.connectToMongoDB(): MongoDatabase {
     val databaseName = environment.config.tryGetString("db.mongo.database.name") ?: "PartyConnections"
 
     val credentials = user.let { userVal -> password?.let { passwordVal -> "$userVal:$passwordVal@" } }.orEmpty()
-    val uri = "mongodb://$credentials$host:$port/?replicaSet=rs0&maxPoolSize=$maxPoolSize&w=majority"
+    val uri = "mongodb://$credentials$host:$port/?replicaSet=rs0&maxPoolSize=$maxPoolSize&w=majority&connectTimeoutMS=1500"
 
-    val mongoClient = MongoClients.create(uri)
+//    val mongoClient = MongoClients.create(uri)
+    val mongoClient = MongoClients.create(
+        MongoClientSettings.builder()
+            .applyConnectionString(ConnectionString(uri))
+            .applyToSocketSettings { builder ->
+                builder
+                    .connectTimeout(1, TimeUnit.SECONDS)
```

Comment:
**suggestion:** Make database connection timeout values configurable

The timeout values for database connections are currently hard-coded. Consider making these configurable, either through environment variables or a configuration file, to allow for easier adjustments in different environments without code changes.